Directory.Build.props

```diff
@@ -86,17 +86,21 @@
     <MicrosoftGraphVersion>4.36.0</MicrosoftGraphVersion>
     <MicrosoftGraphBetaVersion>4.57.0-preview</MicrosoftGraphBetaVersion>
     <MicrosoftExtensionsHttpVersion>3.1.3</MicrosoftExtensionsHttpVersion>
-    <MicrosoftIdentityAbstractions>6.0.0</MicrosoftIdentityAbstractions>    
+    <MicrosoftIdentityAbstractions>6.0.0</MicrosoftIdentityAbstractions>
+    <!--CVE-2024-30105-->
+    <SystemTextJsonVersion>8.0.4</SystemTextJsonVersion>
+    <!--CVE-2023-29331-->
+    <SystemFormatsAsn1Version>8.0.1</SystemFormatsAsn1Version>
   </PropertyGroup>
 
```

Comment:
Only difference between this PR and #2929 is that I removed all .net9 changes since we don't have .net9 in rel/v2.